### ISSUE TITLE
[Suggestion] Shrink Ray

### ISSUE THREAD

**Issue by nuklearcellphoneg** [d3athrow/vgstation13#24610]
#### Description of issue

Hey you know how on vore people can be shrunk down? We should have that as an antag item. If you're shrunk you have no push priority, and if stepped on it breaks all your bones and organs.

In order to obtain the item, make it a requirement to feed into the old RnD deconstructor: Three unlocked Syndicate PDAs that both still have their full forty crystals. This would make it difficult to obtain and also strengthen the power of syndicate code words

Also whenever it's made in-round, a simple check of how they came to pool those 120 crystals together will be able unearth dirty metacommunicators

**Comment by utilitas-** [comment]
[duke nukem squish sound]
The first paragraph is fine.
What the fuck happened from the second? Are you okay?

**Comment by nuklearcellphoneg** [comment]
1. It's fetish material

2. I think it sounds OP and it's literally someone's fetish. I don't want people masturbating to an antagonist who is simply collecting his valids.

**Comment by RealestEstate** [comment]
it's good to criticize your own creations, that how we learn and grow.

**Comment by nuklearcellphoneg** [comment]
So that's why my mom told me I was a mistake... just criticizing me... ty

**Comment by utilitas-** [comment]
\>it's someone's fetish, therefore we should make it basically unacquirable and incredibly not worth it even if you did acquire it through anything but adminbus
Better remove **all** of the potential fetish content, including: vox, balloons, shoes, feet, amputations, vox, cuffs, beds, geometric tiles, vox, multiple antagonists including the changeling and the vampire, kitty ears (which should still give you brain damage), xenomorphs, slime girls, vox, greys, catbeasts...

**Comment by Killette2** [comment]
I'm into breathplay

**Comment by nuklearcellphoneg** [comment]
Yeah but in this case it's an actual fetish from a fetish server.

**Comment by utilitas-** [comment]
https://ss13.booru.org/index.php?page=post&s=view&id=721
Behold, original /vg/station content from a non-fetish server.

**Comment by Kurfursten** [comment]
I actually am of the opinion that a shrink ray is classic sci-fi, like the Invisible Man (our invisibility spray) or The Fly monsters (like TG's teleporting mistakes).

That being said, from reports we've received, the VORE implementation is fucking _terrible_. Apparently being tiny is an advantage because it makes you harder to click, but meanwhile has no trouble operating machinery, using computers, walking around near normal humans, handling a gun, not being murdered by Runtime, etc. In practice, an actual shrink ray would probably do something akin to turn someone into a mouse, just resprited as their human downscaled.
>can no longer pick up items, interact with machines, attack people
>at risk from animals, mouse traps
>can be picked up, ventcrawl

If someone wanted to code a shrink ray the right way I would be all for it, but I'm guessing we would end up with a lazy implementation where it's actually a powergame strat to be 5% size or whatever the minimum is.

**Comment by Exxion** [comment]
Making it work properly would be a tremendous amount of effort to support one weird niche item

**Comment by SGuivenne** [comment]
As much as I love this concept and flagrantly disregard balance, from what I've experienced in at least one adminbus event where sprite shrinking was a thing that was used, there will ALWAYS be at least one person who uses this shit to outright grief people with impunity.  It is absolutely fucking annoying.  Don't do it please.

**Comment by Killette2** [comment]
You could always give shrunken people an invisible regular sized hitbox behind them

**Comment by Exxion** [comment]
Hitbox masks tend to be obnoxious except when they're filling an area that's already surrounded by visible pixels
If you're trying to click something else on the tile, you have to look at the bottom of the screen to see whether or not you'll hit it

**Comment by Kurfursten** [comment]
Don't some servers have a namebox that appears when you hover a person?

Seems good, even outside of the context of unintuitive hitbox masking

**Comment by nuklearcellphoneg** [comment]
I was just thinking that

>If you're shrunk you have no push priority, and if stepped on it breaks all your bones and organs.

That would be balance enough. Also slow their walk/run speed considerably(to match the fact that their stride is about a centimeter long).

**Comment by Kurfursten** [comment]
>can still hold a full size egun and stun down a whole room by being tiny while no one can even pick out your sprite between wires, pipes, and shit on the floor

It should limit victims to picking up only tiny-class items and fail dexterity checks (needed to use complex machinery and computers - monkeys and people wearing boxing gloves are other examples).

**Comment by nuklearcellphoneg** [comment]
Then we need to code 9 mini tiles to exist on each tile, and build a tiny rage cage

**Comment by SECBATON-GRIFFON** [comment]
> Hey you know how on vore

stopped reading there

**Comment by nuklearcellphoneg** [comment]
> > Hey you know how on vore
> 
> stopped reading there

I don't think my joke suggestion from six years ago is going to be taken seriously either.

**Comment by Kurfursten** [comment]
The appropriate response here would be to wait 6 years to reply to SECBATON
